sys/can/conn/isotp.c

```diff
@@ -69,15 +69,6 @@ static inline void get_msg(conn_can_isotp_t *conn, msg_t *msg)
 #endif
 }
 
-static inline int try_get_msg(conn_can_isotp_t *conn, msg_t *msg)
-{
-#ifdef MODULE_CONN_CAN_ISOTP_MULTI
-    return mbox_try_get(&conn->master->mbox, msg);
-#else
-    return mbox_try_get(&conn->mbox, msg);
-#endif
-}
```

Comment:
What's the reasoning behind removing this?

Reply:
This check is already done around the place where this is called (See line 369). This function adds nothing but 1) dead code, 2) a useless extra push to the stack.
At first I tried to rewrite the code so that the check was only done once, but then only a wrapper function was left behind... It made far more sense to me to just change the function call according to the preprocessor defines.

Reply:
Ah I didn't realize this was called from an `#else` block already.